packages/components/src/components/form-submit-button/form-submit-button.tsx

```diff
@@ -1,23 +1,36 @@
-import classNames from 'classnames';
-import PropTypes from 'prop-types';
 import React from 'react';
+import classNames from 'classnames';
 import Button from '../button/button';
 import Text from '../text';
 
+type FormSubmitButton = {
+    className: string;
+    has_cancel: boolean;
+    is_absolute: boolean;
+    is_center: boolean;
+    is_disabled: boolean;
+    label: string;
+    cancel_label: string;
+    form_error: string;
+    cancel_icon: React.ReactElement;
+    is_loading: boolean;
+    onCancel: () => void;
+};
+
 const FormSubmitButton = ({
     cancel_label,
     className,
-    has_cancel,
-    form_error,
+    has_cancel = false,
+    form_error = '',
     cancel_icon,
-    is_center,
-    is_disabled,
-    is_absolute,
+    is_center = false,
+    is_disabled = false,
+    is_absolute = false,
     is_loading,
     label,
     onCancel,
     ...props
-}) => {
+}: Partial<FormSubmitButton>) => {
```

Comment:
shouldnt we specify label as a required prop for the button?

Reply:
It seems that 'text' prop for Button component is optional, and we are passing here 'label' to 'text', so I believe it also should be optional.
thanks for this comment, it makes me to check types again, and I extended prop type for component